src/app/models/constants.ts

```diff
@@ -29,6 +29,7 @@ export class RouteEndpoints {
     static readonly ATTENDANCE = 'Attendance'
     static readonly ATTENDANCE_GET_PARTICIPANTS = 'Attendance/GetParticipants'
     static readonly DASHBOARD_EVENTS = 'Event/GetDashboardEvents'
+    static readonly PARTICIPANTS_LIST = 'User/GetParticipantsForEvent'
```

Comment:
remove this, make use of ATTENDANCE_GET_PARTICIPANTS = 'Attendance/GetParticipants'